addons/website/static/tests/tours/o_editable.js

```diff
@@ -0,0 +1,28 @@
+/** @odoo-module **/
+
+import tour from "web_tour.tour";
+import wTourUtils from "website.tour_utils";
+
+tour.register("test_o_editable", {
+    test: true,
+    url: "/",
+}, [
+    ...wTourUtils.clickOnEditAndWaitEditMode(),
+    wTourUtils.dragNDrop({
+        id: "s_company_team",
+        name: "Team",
+    }),
+    {
+        content: "Drag and drop the Text Highlight building block next to the Team block media.",
+        trigger: "#oe_snippets .oe_snippet[name='Text Highlight'] " +
+            ".oe_snippet_thumbnail:not(.o_we_already_dragging)",
+        extra_trigger: "body.editor_enable.editor_has_snippets",
+        moveTrigger: ".oe_drop_zone",
+        run: "drag_and_drop #wrapwrap .s_company_team .o_editable_media",
+    }, {
+        content: "Verify that the Text Highlight building block isn't next to an o_editable_media.",
+        trigger: ".s_company_team:has(.s_text_highlight):not(:has(" +
+            ".o_editable_media + .s_text_highlight, .s_text_highlight + .o_editable_media)) ",
```

Comment:
I would rather write `.s_company_team :not(.o_not_editable) > .s_text_highlight`, it says the same things with less and more efficient selector: "the text highlight snippet ended up in a column which is not not-editable, in the company team snippet".

And to make sure there was a o_not_editable item, I would adapt the drag and drop above: instead of 
```javascript
        run: "drag_and_drop #wrapwrap .s_company_team .o_editable_media",
```
do
```javascript
        run: "drag_and_drop #wrapwrap .s_company_team .o_not_editable > .o_editable_media",
```

I would also add a step before that drag and drop to check that `#wrapwrap .s_company_team .o_not_editable > .o_editable_media` is actually a valid selector. Otherwise the drag and drop util fallbacks on dropping in the body, in which case your test could start working by chance (as it would probably end up in the company team snippet anyway).